bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/Triboson_ZPrimeToZRToZWW/Triboson_M_R/Triboson_M_R_customizecards.dat

```diff
@@ -0,0 +1,6 @@
+set param_card kkinputs 5 <MASS>
+set param_card kkinputs 2 6.708000e+00
```

Comment:
Which param is this? kkinputs 2 ? Maybe I misunderstood.

Reply:
It is one parameter in the coupling.